src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -268,6 +279,8 @@ protected void onStartup() {
         validateSequenceDictionaries();
 
         checkToolRequirements();
+
+        progressMeter = new ProgressMeter(recordsPerProgressUpdate);
```

Comment:
After seeing how different the speed of the progress meter is between read and variant walkers, I am actually going to switch to a time-based model.